product-web-app/service/authService.ts

```diff
@@ -37,13 +38,19 @@ export default class AuthService {
     }
   }
 
-  async login(user: User): Promise<void> {
+  async login(user: User, res: Response): Promise<void> {
     try {
-      const response = await axios.post('http://localhost:8080/api/login', user);
-
-      return response.data;
-    } catch (e) {
-      throw new Error('Could not login');
+      const apiResponse = await axios.post(API.LOGIN, user);
+      const token = apiResponse.data;
+
+      if (token) {
+        res.cookie('token', token, { httpOnly: true });
```

Comment:
set max-age to 1 hour
```suggestion
        res.cookie('token', token, { httpOnly: true });
```